octoprint_mrbeam/software_update_information.py

```diff
@@ -515,12 +516,12 @@ def _generate_config_of_beamos(plugin, moduleconfig, beamos_version, tierversion
     sorted_config_for_beamos_versions = sorted(
         config_for_beamos_versions.items(),
         key=lambda com: VersionComparator.get_comparator(
-            com[0], plugin.COMPARISON_OPTIONS
+            com[0], COMPARISON_OPTIONS
         ).priority,
     )
 
     config_for_beamos = get_config_for_version(
-        beamos_version, sorted_config_for_beamos_versions, plugin.COMPARISON_OPTIONS
+                                                                                                                                                                                                                    beamos_version, sorted_config_for_beamos_versions, COMPARISON_OPTIONS
```

Comment:
there is something messy with this line, please check

Reply:
done